docs/Internal Systems/Research & Findings/StaticWebGeneratorsManveen.md

```diff
@@ -0,0 +1,35 @@
+# Static Website generators – Manveen 
+
+<br>
+
+There are many static website generators like Gatsby, Jekyll, Hugo, WordPress, Next.js, Eleventy and VuePress. However, since Gatsby and Jekyll are considered at the forefront technologies, an alternative that can be used instead of those is Next.js. 
+
+<br>
+
+## Pros: 
+
+ * One of the pros that Next.js has is that the user may choose whether to prerender the web pages using Static Site Generation or Server-Side Rendering. This feature is more flexible than Gatsby. Additionally, this is especially beneficial if the user needs to scale the project in the future but don't want to commit to a specific size in the beginning. 
+
+* Comparing the payload optimization and progressive web app of Next.js, Gatsby and Jekyll. Gatsby tops all the three, however, if compared Next.js and Jekyll, then Next.js has a better payload optimization and progressive web app than Jekyll.
+
+* Talking of the Maintainability and Debuggability, all three stands at the same position. 
+
+* However, when compared with respect to Declarative rendering and Modern development practices, Gatsby and Next.js tops in all charts and Jekyll lacks that. Below is a chart showing the comparison. 
+
+* Another advantage of Next.js over Jekyll is Design. Jekyll lacks Programmatic design, Design systems and component libraries whereas Gatsby and Next.js has it all. 
+
+* Lastly, when compared with respect to documentation, and accessibility, Next.js competes well with Gatsby and Jekyll.  
+<br>
+## Cons: 
+<br>
+* However, the main distinction is that, while Next.js requires a server to work (in order to provide SSR capabilities), Gatsby and Jekyll can run without one. Also, Gatsby provides pure HTML/JS/CSS at build time, whereas Next.js generates HTML/JS/CSS at runtime. 
+
+<br>
+
+## [Infographic of comaprison](https://www.gatsbyjs.com/features/jamstack/gatsby-vs-nextjs-vs-jekyll) 
```

Comment:
Please link the url in a sentence, not a heading, and provide a sentence explaining the link.
This is an accessibility change.

Reply:
Fixed this is now no longer a heading.